### DISCUSSION TITLE
关于罗技ghub的IsMouseButtonPressed方法触发问题

### DISCUSSION THREAD

**Discussion by xiangchidadidou** [community/community#76381] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

写罗技脚本时候如果想要完成鼠标按键按下的监听操作，会发现除了鼠标左键能够持续触发IsMouseButtonPressed方法以外，其他的按键都不能触发，代码如下
if IsStart then
    if IsMouseButtonPressed(4) then --此处的方法并不会触发（如果换成1，则可以）
      repeat
        PressKey("q")
        Sleep(50)
        ReleaseKey("q")
      until not IsMouseButtonPressed(4)
    end
  end

有没有大佬教我一下

**Comment by xiangchidadidou** [comment]
我写了一个更为直观的代码，这里的 IsMouseButtonPressed(1)能触发，但是 IsMouseButtonPressed(2)就无法触发了
EnablePrimaryMouseButtonEvents(true)

function OnEvent(event, arg)
    if IsMouseButtonPressed(1) then
        OutputLogMessage("111 \n")
    end
end

**Comment by Akash1134** [comment]
Hi @xiangchidadidou , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
